website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -88,6 +88,57 @@ Registering the external services with Consul is a multi-step process:
 
 ### Register external services with Consul
 
+There are two ways to register an external service with Consul:
+1. If you have [`TransparentProxy`](/docs/k8s/helm#v-connectinject-transparentproxy) enabled, you can declare external endpoints with the [Destination](/docs/connect/config-entries/service-defaults#terminating-gateway-destination) field of `service-defaults`.
```

Comment:
```suggestion
1. If [`TransparentProxy`](/docs/k8s/helm#v-connectinject-transparentproxy) is enabled, you can declare external endpoints in the [`Destination`](/docs/connect/config-entries/service-defaults#terminating-gateway-destination) field of `service-defaults`.
```
The docs for the `destination` field are still in production, correct?

Reply:
Those docs are not merged but already approved here: https://github.com/hashicorp/consul/pull/13950 
I'll wait until those get merged before merging this one.